BlockSettleSigner/main.cpp

```diff
@@ -316,14 +319,22 @@ static int QMLApp(int argc, char **argv
       adapter.setCloseHeadless(settings->closeHeadless());
 
       QMLAppObj qmlAppObj(&adapter, logger, settings, splashScreen, engine.rootContext());
-      QTimer::singleShot(0, &qmlAppObj, &QMLAppObj::Start);
 
       switch (settings->runMode()) {
       case bs::signer::ui::RunMode::fullgui:
          engine.load(QUrl(QStringLiteral("qrc:/qml/main.qml")));
          break;
       case bs::signer::ui::RunMode::litegui:
          engine.load(QUrl(QStringLiteral("qrc:/qml/mainLite.qml")));
+
+         terminalConnectionTimer.setSingleShot(true);
+         //http://185.213.153.35:8081/browse/BST-2786
```

Comment:
I'm guessing anyone can access this IP, scrub that out from the comment, just leave teh BST code